CIME/SystemTests/system_tests_common.py

```diff
@@ -516,7 +516,7 @@ def _get_mem_usage(self, cpllog):
         """
         memlist = []
         meminfo = re.compile(
-            r".*model date =\s+(\w+).*memory =.*highwater\)\s+(\d+\.?\d+).*usage"
+            r".*model date =\s+(\w+).*memory =\s+(\d+\.?\d+).*highwater"
```

Comment:
@amametjanov , this change looks unrelated to this PR. Are you sure you want to include this?

Reply:
I was wondering why this was showing here. The HEAD of the branch was E3SM's CIME hash 98106c908bd3b5514239e56efdc7ab6a53524065.
I just rebased the branch onto latest CIME master and this diff is gone. I hope this is okay now.